docker-compose.yml

```diff
@@ -50,6 +50,7 @@ services:
       AIRFLOW_CONN_BOUWSTROOMPUNTEN_PASSWD: ${dataservices_bouwstroompunten_passwd}
       AIRFLOW_CONN_BOUWSTROOMPUNTEN_USER: ${dataservices_bouwstroompunten_user}
       AIRFLOW_CONN_BOUWSTROOMPUNTEN_BASE_URL: "https://maps.nrgaccounting.nl"
+      AIRFLOW_CONN_RIVM_BASE_URL: "https://data.rivm.nl/covid-19/"
```

Comment:
Is deze ENV VAR nodig? Die moet dan ook weer in de Openstack  config worden opgenomen>
Kan dat niet gewoon in hardcoded in het shellscript?

Reply:
Ja goeie dat kon ook, dat zou sneller zijn. Maar dacht dat dit beter zou zijn. Ik heb het inmiddels laten opnemen in Openstack.